content/guides/overview/why-cypress.md

```diff
@@ -114,10 +114,17 @@ do that no other testing framework can:
 - **Consistent Results:** Our architecture doesn’t use Selenium or WebDriver.
   Say hello to fast, consistent and reliable tests that are flake-free.
 - **Screenshots and Videos:** View screenshots taken automatically on failure,
-  or videos of your entire test suite when run from the CLI.
+  or videos of your entire test suite when run from the CLI. Record to the
+  Dashboard to store them with your test results for easy online browsing.
```

Comment:
maybe something like "... for zero-configuration debugging" or "out-of-the-box debugging" - idk, something around "debugging" rather than just "browsing"